src/plugins/condition/components/ConditionCollection.vue

```diff
@@ -28,25 +28,34 @@
                 <span class="c-cs-button__label">Add Condition</span>
             </button>
         </div>
-        <div class="condition-collection">
-            <div v-for="conditionIdentifier in conditionCollection"
-                 :key="conditionIdentifier.key"
-                 class="conditionArea"
-            >
-                <div v-if="isEditing">
-                    <ConditionEdit :condition-identifier="conditionIdentifier"
+        <div class="c-c condition-collection">
+            <ul class="c-c__container-holder">
+                <li v-for="(conditionIdentifier, index) in conditionCollection"
+                    :key="conditionIdentifier.key"
+                >
+                    <div v-if="isEditing">
+                        <div class="c-c__drag-ghost"
+                             @drop.prevent="dropCondition"
+                             @dragenter="dragEnter"
+                             @dragleave="dragLeave"
+                             @dragover.prevent
+                        ></div>
+                        <ConditionEdit :condition-identifier="conditionIdentifier"
+                                       :current-condition-identifier="currentConditionIdentifier"
+                                       :condition-index="index"
+                                       @update-current-condition="updateCurrentCondition"
+                                       @remove-condition="removeCondition"
+                                       @condition-result-updated="handleConditionResult"
+                                       @set-move-index="setMoveIndex"
```

Comment:
You don't need this `set-move-index` event.
In ConditionEdit you can set the startIndex by doing this:
`e.dataTransfer.setData("moveIndex", this.conditionIndex.toString());`

And then retrieve it in ConditionCollection.dropCondition by doing this:
`let moveIndex = e.dataTransfer.getData('startIndex');`
Then you can cast it to a Number if needed.

Reply:
N/A - leave as is for now.

Reply:
@JoelMcKinnon Please use camelCase for event names for consistency. There are only couple of places in the source code (flexible layout and date picker) where it's not done that way, which we'll need to update at some point.

Reply:
VueJS docs say you should always use kebab-case: https://vuejs.org/v2/guide/components-custom-events.html#Event-Names

I tried changing to camelCase and the events didn't work.

Reply:
They are recommending it,  but it doesn't mean camelCase event names don't work.

We need to discuss this with the team to decide which way to go.